images/acs_engine_build/build_acs.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/bash
+
+set -x
+set -e
+
+set -o pipefail
+
+# Global vars
+REPO=${REPO:-"http://github.com/Azure/acs-engine"}
+BRANCH=${BRANCH:-"master"}
+
+ACS_DIR=${GOPATH}/src/github.com/Azure/acs-engine
+ACS_BIN="$ACS_DIR/bin/acs-engine"
+
+OUTPUT_DIR=$HOME/output
+ARTIFACTS_DIR=$OUTPUT_DIR/artifacts
```

Comment:
Why do we need this?

Reply:
we need it for the junit_runner.xml, so we can parse it

Reply:
really? and where do you parse it? Who exactly outputs junit_runner.xml if you're not running kubetest inside this job?

Reply:
This is the script for the job that builds and uploads acs-engine right?

Reply:
yeah, though initially that it was in the other script, my bad